### PR TITLE
Fixed missing labelFont property, added Swift version in project settings

### PR DESCRIPTION
**Pull request by imryan**
After importing the pod into my Obj-C project, `labelFont` was being left out of the `TriLabelView-Swift.h` file.

### CODE REVIEW

File: TriLabelView.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -437,6 +437,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 3.0;
```
Comment (mukeshthawani) [inline TriLabelView.xcodeproj/project.pbxproj]: Can we change this to Swift 4.0?

File: TriLabelView/TriLabelView.swift [hunk 2]
```diff
@@ -80,7 +80,11 @@ import UIKit
     }
     
     /// The font for the label text.
-    public var labelFont: UIFont = UIFont(name: "HelveticaNeue-Bold", size: 20)!
+    @IBInspectable public var labelFont: UIFont = UIFont.systemFont(ofSize: 15, weight: UIFontWeightBold) {
```
Comment (mukeshthawani) [inline TriLabelView/TriLabelView.swift]: You need to change this to UIFont.Weight.bold as `'UIFontWeightBold' has been renamed to 'UIFont.Weight.bold'` 
https://travis-ci.org/mukeshthawani/TriLabelView/builds/324846428?utm_source=github_status&utm_medium=notification

Comment (imryan) [reply TriLabelView/TriLabelView.swift]: Ah, gotcha. Let me edit.

File: TriLabelView.xcodeproj/project.pbxproj (hunk 1 above)
Comment (imryan) [reply TriLabelView.xcodeproj/project.pbxproj]: Yes sir :)

**Comment by imryan** [comment]
What's up with this conflict?

**Review by mukeshthawani — APPROVED** [review]

**Comment by imryan** [comment]
😄

**Comment by mukeshthawani** [comment]
Good job! 👍

**Comment by imryan** [comment]
Haha, cheers man. Still getting that annoying 'need to specify Swift language version for all targets using Swift...' which is strange. We've set it on all targets in the project, yet when I go to the `Pods.xcodeproj` in my project, and go into build settings for TriLabelView, the lang version still shows 'Unspecified.' Hmm...

**Comment by mukeshthawani** [comment]
Did you try by pointing it to the master?

**Comment by imryan** [comment]
@mukeshthawani I have, same results unfortunately.
 `pod 'TriLabelView', :git => 'https://github.com/mukeshthawani/TriLabelView.git', :branch => 'master'`

**Comment by mukeshthawani** [comment]
Let's continue this discussion on #11

**Pull request merged**